seed/seed.json

```diff
@@ -685,6 +685,34 @@
                 "channel": ["BETA", "NIGHTLY"],
                 "platform": ["MAC"]
             }
+        },
+        {
+            "name": "MacCoreLocationBackendReleaseStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_association": {
+                        "enable_feature": ["MacCoreLocationBackend"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 0,
+                    "feature_association": {
+                        "disable_feature": ["MacCoreLocationBackend"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "92.1.27.109",
+                "channel": ["RELEASE"],
```

Comment:
I think it's fine to add this to the existing study if there were no changes between 91.1.26.74 and 92.1.27.109 for this feature.

Reply:
Right. It's not changed. Updated.